app/javascript/app/components/anchor-nav/anchor-nav-component.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+
+import styles from './anchor-nav-styles.scss';
+
+const AnchorNav = (props) => {
+  const { links } = props;
+  return (
+    <div className={styles.anchorNav}>
+      {links.map((link, index) =>
+        (<NavLink
+          exact={index === 0}
```

Comment:
Why do you only do exact with the first element? I think all of them should be exact https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/Route.md#exact-bool